package.json

```diff
@@ -20,6 +20,7 @@
     "changelog:validate": "yarn workspaces foreach --all --no-private --parallel --interlaced --verbose run changelog:validate",
     "create-component:react": "yarn workspace @metamask/design-system-react create-component",
     "create-package": "ts-node scripts/create-package",
+    "generate-icons:react-native": "yarn workspace @metamask/design-system-react-native ts-node src/components/Icons/Icon/scripts/generate-assets.scripts.ts",
```

Comment:
Given that the SVG build script isn't currently integrated into any build process, it might be beneficial to add it to the `design-system-react-native` build script. This would ensure that icons are built consistently as part of the design system's build process. Not sure of the utility at this level I suppose if we are adding a new icon it may be helpful to run it from root.  What do you think?

Reply:
somehow running `ts-node` in the `design-system-react-native` package isnt work, but doing it through `yarn workspace` works so I added it there